src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -80,6 +80,8 @@ boot_newcomp_sinv_alloc(spdid_t spdid)
 	void *user_cap_vaddr;
 	struct cos_compinfo *interface_compinfo;
 	struct cos_compinfo *newcomp_compinfo = new_comp_cap_info[spdid].compinfo;
+	/* TODO: Purge rest of booter of spdid convention */
+	unsigned long token = (unsigned long)spdid;
```

Comment:
I did this based on your comment on previous commit. The llbooter is infested with spdid, so I figured this felt at least more consistent with current code while acknowledging our intention to switch to token. As opposed to changing the spdid argument for the function all together.

Reply:
OK fine.  I suppose the fires of `spdid` purgatory will remain unsatiated for now.